plugins/src/android/OptimoveInitProvider.java

```diff
@@ -74,4 +93,9 @@ private String getStringConfigValue(String packageName, Resources resources, Str
 
         return resources.getString(resId);
     }
+
+    @Nullable
+    private String handleNullValues(String credentials) {
```

Comment:
minor random stuff. maybe a little neater could combining `handleNullValues` and `getStringConfigValue` to get 

```
private String readConfigValue(String packageName, Resources resources, String key) {
 int resId = resources.getIdentifier(key, "string", packageName);
 if (0 == resId) {
    return null;
 }

  String value = resources.getString(resId);
  return TextUtils.isEmpty(value) ? null : value;
}
```

Reply:
done, I agree it is neater that way